samples/drawing/drawing.cpp

```diff
@@ -1806,7 +1806,7 @@ void MyCanvas::DrawSystemColours(wxDC& dc)
     }
 
     int lineHeight = textSize.GetHeight();
-    wxCoord x(FromDIP(10));
+    wxCoord x(dc.FromDIP(10));
```

Comment:
Maybe we should move this function our of `MyCanvas`, so that it wouldn't be able to use window scale factor accidentally?

Reply:
This applies to all the `MyCanvas::Draw*` functions. I think this is a too big and unrelated change to make in this PR.